web-marketplace/src/components/organisms/LoginButton/LoginButton.tsx

```diff
@@ -124,10 +129,9 @@ const LoginButton = ({ size = 'small' }: Props) => {
     <>
       <div className={styles.root}>
         <>
-          {!isConnected && loaded && (
+          {!isConnected && !activeAccountId && !authLoading && walletLoaded && (
```

Comment:
There are several occurrences of this expression should we make a wrapper hook (like `useAuthData`) that uses both `useAuth` and `useWallet` to return this expression and everything we need at once?